orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/utils/UpdateQueryGeneratorTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.datastore.ogm.orientdb.utils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.hibernate.ogm.model.impl.DefaultEntityKeyMetadata;
+import org.hibernate.ogm.model.key.spi.EntityKey;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * @author Sergey Chernolyas &lt;sergey.chernolyas@gmail.com&gt;
+ */
+public class UpdateQueryGeneratorTest {
+
+	public UpdateQueryGeneratorTest() {
+	}
+
+	@BeforeClass
+	public static void setUpClass() {
+	}
+
+	@AfterClass
+	public static void tearDownClass() {
+	}
+
+	@Before
+	public void setUp() {
+	}
+
+	@After
+	public void tearDown() {
+	}
+
+	/**
+	 * Test of generate method, of class UpdateQueryGenerator.
+	 */
+	@Test
+	public void testGenerate() {
```

Comment:
I think you need to describe better what you are testing here. Maybe add an example of the query you are trying to generated.
Is it the execution of an update query?